stable/spring-cloud-data-flow/templates/server-deployment.yaml

```diff
@@ -0,0 +1,137 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "scdf.fullname" . }}-server
+  labels:
+    app: {{ template "scdf.name" . }}
+    component: server
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: {{ template "scdf.name" . }}
+        component: server
+        release: "{{ .Release.Name }}"
+    spec:
+      containers:
+      - name: {{ template "scdf.fullname" . }}-server
+        image: {{ .Values.server.image }}:{{ .Values.server.version }}
+        imagePullPolicy: {{ .Values.server.imagePullPolicy }}
+        volumeMounts: [
```

Comment:
Any reason why you mix list syntaxes? Why not keep it consistent and do this:

```yaml
- name: data-flow
  mountPath: /etc/secrets/data-flow
  readOnly: true
- name: rabbitmq
  mountPath: /etc/secrets/rabbitmq
  readOnly: true
...
```

Reply:
updated